### ISSUE TITLE
[Bug]: Some errors reported when executing SQL to delete cn/dn pod

### ISSUE THREAD

**Issue by heni02** [matrixorigin/matrixone#8619]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```Markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93):98a464d
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior

When executing the tpch test, separately kill cn and dn pod . Sometimes SQL execution reports errors，are these expected？
kill cn1：ERROR 20500 (HY000) at line 1: rpc timeout
kill cn0: ERROR 2013 (HY000) at line 1: Lost connection to MySQL server during query
kill dn：ERROR 1045 (28000): Access denied for user dump. SQL parser error: table "mo_account" does not exist

job：https://github.com/matrixorigin/mo-nightly-regression/actions/runs/4488303584/jobs/7892816454

Well, two questions need to be identified ：
one is to determine which errors should not be reported in the event of POD unavailability, and whether there are other ways to avoid them. 
The second is to throw out error information so that users can intuitively understand what caused the problem

**Comment by nnsgmsone** [comment]
现在断开连接后可能收到不同模块的错误，可能是前端，可能是proxy，你预期的错误是什么呢？ @heni02 和叶磊说的有一样，这个可能需要产品定义一下此时的行为，然后统一这时候的报错信息。。可能应该作为一个需求。。

**Comment by aressu1985** [comment]
@domingozhang pls confirm the error message.

**Comment by nnsgmsone** [comment]
ok

**Comment by nnsgmsone** [comment]
ok, get it

**Comment by nnsgmsone** [comment]
can move to 0.9? @domingozhang

**Comment by domingozhang** [comment]
will be fixed in 0.9

**Comment by sukki37** [comment]
This issue has been inactive. Closing to keep the tracker clean. Reopen if still relevant. Thanks!
